src/resources/js/tickets-block.js

```diff
@@ -381,10 +381,9 @@ tribe.tickets.block = {
 		const hasOptout = $ticket.has( obj.selector.itemOptOut ).length;
 		if ( hasOptout ) {
 			const $item = $ticket.closest( obj.selector.item );
-			const $optout = $item.find( obj.selector.itemOptOut );
 			if ( 0 < newQuantity ) {
 				//$optout.show();
-				$item.addClass( 'show-optout');
+				$item.addClass( 'show-optout' );
```

Comment:
whitespace

Reply:
Also changed this to utilize a class change instead of show/hide as we needed to reference the state elsewhere.